mmpose/visualization/local_visualizer.py

```diff
@@ -12,6 +12,7 @@
 
 from mmpose.registry import VISUALIZERS
 from mmpose.structures import PoseDataSample
+from simCCVis import SimCCVisualizer
```

Comment:
I suggest renaming this file to `simcc_vis.py` and place it under `mmpose/visualization` folder